NEWS.md

```diff
@@ -6,10 +6,67 @@ Version X.Y of OpenDDS.
 - Add support for versioned namespaces
 - Rework type support implementation in order to expose less internal header
   and classes to user code
+- Add read/take_instance_w_condition
+- Time Based Filter QoS 
+- C++11 Updates
+  - When C++11 is enabled make use of C++11 shared_ptr and to_string instead of using boost 
+  - Added configure and MPC support for C++11 version of ishapes 
+  - When C++11 is enabled use noexcept(false) 
+- Extend OpenDDS type support to provide user-initiated unregistering of types from a
+  domain participant. This has been included as a type support extension because
+  unregister_type is not required by the DDS specification.
+- Removed dependency on the dds4ccm_opendds feature and let configure also add a line
```

Comment:
I must've missed the revert.  Thanks for clarifying.  I'll hold off on this until next release.